pkl/unmarshal_test.go

```diff
@@ -84,6 +84,9 @@ var anies []byte
 //go:embed test_fixtures/msgpack/unknown_type.pkl.msgpack
 var unknownType []byte
 
+//go:embed test_fixtures/msgpack/arrays_too_long.pkl.msgpack
```

Comment:
Everything in the `pkl/test_fixtures/msgpack` dir is generated by `go run cmd/internal/gen-fixtures/gen-fixtures.go`; you should be able to blow away the directory and run that command and get the same output.

Can you add some logic there to also produce this file?

Reply:
This one is hand-tooled so I'll move it to a different path.